src/backend/drm/device/mod.rs

```diff
@@ -372,7 +375,7 @@ impl DrmDevice {
     }
 
     /// Returns the device_id of the underlying drm node
-    pub fn device_id(&self) -> dev_t {
+    pub fn device_id(&self) -> DrmNode {
```

Comment:
We now have `DrmDevice::device_id` that returns `DrmNode` and says in its docs `Returns the device_id of the underlying drm node`, and then `DrmNode` has `dev_id()` that returns `dev_t` and also says in its docs `Returns the device_id of the underlying DRM node.`. 

This is kinda confusing, I believe that we should distinguish between the two, eg. leave `device_id()` as it was and add `device_node()` getter instead, with appropriate docs that it returns a DrmNode, not the device_id

Reply:
Good point! I'll fix that.

Reply:
hm... I'd even go further and remove `device_id` and replace it with `device_node` because you can easily retrieve the ID from the node afterwards.

Reply:
I don't mind

Reply:
done!